conda_smithy/feedstocks.py

```diff
@@ -83,9 +83,9 @@ def clone_feedstock(feedstock_gh_repo, feedstocks_dir):
     clone_directory = os.path.join(feedstocks_dir, repo.name)
     if not os.path.exists(clone_directory):
         print('Cloning {}'.format(repo.name))
-        new_repo = Repo.clone_from(repo.clone_url, clone_directory)
-        clone.delete_remote('origin')
+        Repo.clone_from(repo.clone_url, clone_directory)
     clone = Repo(clone_directory)
+    clone.delete_remote('origin')
```

Comment:
Are we sure this is a good idea? It seems like the remote could potentially be undefined here (was already cloned and remote removed); whereas, before we knew `origin` is defined. If we do move it down here, there should at least be a `try...except`. Would be a `GitCommandError` that we need to catch IIRC.